ansible/local_scripts/goinstall.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+set -e
+
+VERSION="1.8.1"
+
+print_help() {
+    echo "Usage: bash goinstall.sh OPTION"
+    echo -e "\nOPTIONS:"
+    echo -e "  --32\t\tInstall 32-bit version"
+    echo -e "  --64\t\tInstall 64-bit version"
+    echo -e "  --arm\t\tInstall armv6 version"
+    echo -e "  --remove\tTo remove currently installed version"
```

Comment:
You don't need any other options than `64`, b/c Swan is supporting 64 bit platforms only.